scripts/purge_queue.py

```diff
@@ -0,0 +1,109 @@
+import sys
+
+import argparse
+import boto3
+import json
+import more_itertools
+from azul import config
+import logging
+
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+parser = argparse.ArgumentParser(description=__doc__)
+
+subparsers = parser.add_subparsers(help="sub-command help", dest="command")
+
+parser_dump = subparsers.add_parser("dump", help="Dump contents of queue into designated file")
+parser_dump.add_argument("queue_name", help="Name of the queue to obtain the messages from")
+parser_dump.add_argument("file_name", help="Name of file to write output to")
+parser_dump.add_argument("--delete", "-d", action='store_true',
+                         help="Purge messages from the queue after writing them to the designated file. "
+                              "By default the messages will be returned to the queue")
+parser_dump.add_argument("--json-body", "-J", action="store_true", help="Serialize json for queue message body")
+parser_list = subparsers.add_parser("list", help="List all available queues")
+
+sqs = boto3.resource("sqs")
+
+
+def list_queues():
+    prefix = config._index_prefix  # Limit queues displayed to our ES index deployment names.
+    all_queues = [q.url for q in sqs.queues.all()]
+    for queue_url in all_queues:
+        _, _, queue_name = urlparse(queue_url).path.rpartition("/")
+        if queue_name.startswith(prefix):
+            print(queue_name)
+
+
+def get_messages_from_queue(args):
+    queue_name = sqs.get_queue_by_name(QueueName=args.queue_name).url
+    output_file_name = args.file_name
+    queue_resource = sqs.Queue(url=queue_name)
+    logger.info("Obtaining messages from queue '%s'", queue_name)
+    logger.info("Direct output '%s'", output_file_name)
+
+    five_minutes_in_seconds = 60 * 5
+    logger.info("Default visibility timeout '%i'", output_file_name)
+    list_of_messages = []
+    while True:
+        messages = queue_resource.receive_messages(
+            AttributeNames=['All'],
+            MaxNumberOfMessages=10,
+            VisibilityTimeout=five_minutes_in_seconds)
+
+        if len(messages) == 0:  # Nothing left in queue
+            break
+        else:
+            list_of_messages.extend(messages)
+    with open(output_file_name, "w") as f:
+        queue_messages = json.dumps({"queue": queue_name, "messages": extract_fields(args.json_body, list_of_messages)})
+        f.write(str(queue_messages))
+    logger.info("Total messages handled: %i", len(list_of_messages))
+    message_batches = list(more_itertools.chunked(list_of_messages, 10))
+    if args.delete:
+        logger.info("Removing messages from `%s` queue", queue_name)
```

Comment:
```suggestion
        logger.info("Removing messages from queue '%s'", queue_name)
```

Remove all backqoutes from the source.